### PR TITLE
make Pcb.write fail if the connection is reset

### PR DESCRIPTION
**Pull request by samoht**

### CODE REVIEW

File: tcp/flow.ml [hunk 1]
```diff
@@ -35,57 +40,54 @@ module Make(IP:V1_LWT.IP)(TM:V1_LWT.TIME)(C:V1.CLOCK)(R:V1.RANDOM) = struct
     | `Refused
   ]
 
+  let err_timeout () =
+    (* Printf.printf "Failed to connect to %s:%d\n%!" *)
+    (* (Ipaddr.V4.to_string daddr) dport; *)
```
Comment (avsm) [inline tcp/flow.ml]: Can't this use the `Log` module?

File: tcp/flow.ml [hunk 2]
```diff
@@ -14,7 +14,12 @@
  * OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
  *)
 
-open Lwt
+let (>>=) = Lwt.(>>=)
```
Comment (avsm) [inline tcp/flow.ml]: I think you can `open Lwt.Infix` to get these operators.

File: tcp/flow.ml (hunk 1 above)
Comment (samoht) [inline tcp/flow.ml]: the log module is not merged yet, but yes that's the plan :-)

File: tcp/flow.ml (hunk 2 above)
Comment (samoht) [inline tcp/flow.ml]: I never know in which version of `Lwt` the `Infix` module appeared. Thanks for the suggestion, I'll use that from now on.

**Pull request merged**